backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -105,10 +105,12 @@ def reshape_dataset_for_curation_api(dataset: dict, preview=False) -> dict:
         else:
             dataset[ontology_element] = []
 
-    if value := dataset.pop("name", None):
-        dataset["title"] = value
-    if value := dataset.pop("is_primary_data", None):
-        dataset["is_primary_data"] = is_primary_data_mapping.get(value)
+    if not preview:  # Add these fields only to full (and not preview) Dataset metadata response
+        dataset["revision_of"] = dataset.pop("original_id", None)
```

Comment:
Why only in the preview?

Reply:
you mean why _not_ in the preview...?

None of these attributes are supposed to be returned on the preview response object.

Reply:
We include revision_of in the collection portion. If a response returns information about the collection `revision_of`, it makes sense to also return information about the dataset `revision_of`